kodilla-hibernate/src/main/java/com/kodilla/kodilla/hibernate/invoice/Item.java

```diff
@@ -0,0 +1,90 @@
+package com.kodilla.kodilla.hibernate.invoice;
+
+import com.sun.istack.NotNull;
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "ITEMS")
+public class Item {
+
+    private int id;
+    private Product product;
+    private Invoice invoice;
+    private BigDecimal price;
+    private int quantity;
+    private BigDecimal value;
+
+    public Item() {
+    }
+
+    public Item(Product product, Invoice invoice, BigDecimal price, int quantity, BigDecimal value) {
```

Comment:
czy value potrzebne w tym konstruktorze? Przeciez mozesz sobie to wyliczyc:
this.value = price * quantity (oczywiści jako BigDecimal musisz to zrobić)